netmanager/src/views/components/DataDisplay/DeviceView/DeviceEdit.js

```diff
@@ -33,6 +33,12 @@ const EDIT_OMITTED_KEYS = [
   "pictures",
 ];
 
+const AccordionDetails = styled(MuiAccordionDetails)(({ theme }) => ({
+    position: 'relative',
```

Comment:
This is meant to reduce the height of the accordion as requested by the platform team
![image](https://user-images.githubusercontent.com/41787587/159912741-5a3c0705-60f7-438d-aa9a-62f2c95baf7b.png)